cloud/helpers.js

```diff
@@ -0,0 +1,126 @@
+/*
+  Computes a unit reward value in Wei using the formula:
+  toWei(maxReward) / maxTaskers - 1
+*/
+async function computeUnitRewardWei(maxReward, maxTaskers) {
+  const maxRewardWei = Number(
+    await Moralis.Cloud.units({
+      method: "toWei",
+      value: maxReward,
+    })
+  );
+  const unitRewardWei = maxRewardWei / maxTaskers - 1; // ensures that unitRewardWei*maxTaskers < maxRewardWei
```

Comment:
Just a note that the discrepancy between this and the smart contract is here, where the unitRewardWei was calculated without the -1 in the smart contract. @jennsun let's modify this to make the two consistent

Reply:
Shouldn't this cause a discrepancy of 1 Wei = 10^-18 MATIC?